cli/cdsctl/workflow_log.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+)
+
+var (
+	workflowLogCmd = cli.Command{
+		Name:    "logs",
+		Aliases: []string{"log"},
+		Short:   "Manage CDS Workflow Run Logs",
+		Long: `Download logs from a workflow run.
+
+	# list all logs files
+	$ cdsctl workflow logs list KEY WF 1
+	
+	# download all logs files
+	$ cdsctl workflow logs download KEY WF 1
+	
+	# download only one file:
+	$ cdsctl workflow logs download KEY WF 1 WF-1.0-pipeline.myPipeline-stage.MyStage-job.MyJob-status.Success-step.0.log
+
+`,
+	}
+
+	workflowLog = cli.NewCommand(workflowLogCmd, nil,
+		[]*cobra.Command{
+			cli.NewCommand(workflowLogListCmd, workflowLogListRun, nil, withAllCommandModifiers()...),
+			cli.NewCommand(workflowLogDownloadCmd, workflowLogDownloadRun, nil, withAllCommandModifiers()...),
+		})
+)
+
+var workflowLogListCmd = cli.Command{
+	Name:  "list",
+	Short: "List logs from a workflow run",
+	Long: `List logs from a workflow run. There on log file for each step.
+
+	# list all logs files from projet KEY, with workflow named WD on run 1
+	$ cdsctl workflow logs list KEY WF 1
+
+`,
+	Ctx: []cli.Arg{
+		{Name: _ProjectKey},
+		{Name: _WorkflowName},
+	},
+	Args: []cli.Arg{
+		{Name: "number"},
+	},
+}
+
+func workflowLogListRun(v cli.Values) error {
+	number, err := strconv.ParseInt(v["number"], 10, 64)
```

Comment:
if you define the `number` arg as kind `reflect.Int64`, you should use `v.GetInt64("number")`, to avoid this boilerplate code